ci/check_gucs_are_alphabetically_sorted.sh

```diff
@@ -5,6 +5,6 @@ set -euo pipefail
 source ci/ci_helpers.sh
 
 # extract citus gucs in the form of "citus.X"
-grep -o -E "(\.*\"citus.\w+\")," src/backend/distributed/shared_library_init.c > gucs.out
+grep -o -E "(\.*\"citus\.\w+\")," src/backend/distributed/shared_library_init.c > gucs.out
```

Comment:
Can we have this (and gucs.out) file changes in a separate PR?

Also, btw why do we both gucs.out and ci/gucs.out?